axelrod/tournament.py

```diff
@@ -1,11 +1,12 @@
 """Recreate Axelrod's tournament."""
 
 import inspect
-import itertools
+import numpy
```

Comment:
I know this makes things a lot easier but I really would like do like that the base tournament only requires base python (I know the analysis uses numpy and matplotlib)... A quick scan shows that you're using this to create an empty matrix, perhaps you could throw in an exception and do that another way if need be?